gcp/compute/helpers.py

```diff
@@ -1,3 +1,36 @@
+def does_firewall_open_all_ports_to_any(firewall):
+    """
+    Returns True if firewall has a rule to open all ports to any source. Excludes ICMP.
+
+    >>> does_firewall_open_all_ports_to_any({})
+    False
```

Comment:
Does GCP default to closed? 

I think AWS default to all outbound open.

Reply:
@g-k You mean if no rules are present?

Reply:
From https://cloud.google.com/vpc/docs/firewalls#default_firewall_rules - the networks have default deny all.